multinet/uploaders/csv.py

```diff
@@ -84,8 +92,25 @@ def upload(
     except csv.Error:
         raise CSVReadError()
 
+    # TODO: This temporarily needs to be done here, so that validation of the metadata
+    # can be done before the table is actually created. Once the API is updated, this
+    # will change.
```

Comment:
If you file an issue we can track this both here and in GitHub.

Reply:
https://github.com/multinet-app/multinet-server/issues/493